pkg/monitor/monitor/template.go

```diff
@@ -364,7 +390,8 @@ func scrapeJob(jobName string, componentPattern config.Regexp, cmodel *MonitorCo
 				},
 			},
 		}
-		if cmodel.EnableTLSCluster {
+
+		if cmodel.EnableTLSCluster && jobName != "dm-master" && jobName != "dm-worker" {
```

Comment:
Please define a constant for the `dm-master/dm-worker` because the string literal has presented at multiple places.

Reply:
Please also define a function for the dm jobs which is used for several places:
```
func isDMJob(jobName string) bool {
  return jobName == "dm-master" || jobName == "dm-worker"
}
```

Reply:
fixed